.pre-commit-config.yaml

```
@@ -1,10 +1,20 @@
 repos:
+  -   repo: https://github.com/pycqa/isort
+      rev: 5.10.1
+      hooks:
+      - id: isort
+        language_version: python3
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.31.0
+    hooks:
+      - id: pyupgrade
+        args:
+          - --py38-plus
```

Since we still support Python 3.7 today

```suggestion
          - --py37-plus
```